_bibliography/references.bib

```diff
@@ -28,37 +28,112 @@
 
 
 
-@inproceedings{CJJMST21,
+@inproceedings{BJJSV22,
   category  = {using},
-  author    = {Murat Cubuktepe and
+  author    = {Thom S. Badings and
                Nils Jansen and
                Sebastian Junges and
-               Ahmadreza Marandi and
-               Marnix Suilen and
-               Ufuk Topcu},
-  title     = {Robust Finite-State Controllers for Uncertain POMDPs},
-  booktitle = {{AAAI}},
-  pages     = {11792--11800},
-  publisher = {{AAAI} Press},
-  year      = {2021}
+               Mari{\"{e}}lle Stoelinga and
+               Matthias Volk},
+  title     = {Sampling-Based Verification of {CTMCs} with Uncertain Rates},
+  booktitle = {{CAV} {(2)}},
+  series    = {Lecture Notes in Computer Science},
+  volume    = {13372},
+  pages     = {26--47},
+  publisher = {Springer},
+  year      = {2022},
+  doi       = {10.1007/978-3-031-13188-2\_2},
+  arxiv-id  = {2205.08300},
+  material  = {https://doi.org/10.5281/zenodo.6523863},
+}
+
+@inproceedings{BVKKS22,
+  category  = {feature},
+  author    = {Daniel Basg{\"{o}}ze and
+               Matthias Volk and
+               Joost{-}Pieter Katoen and
+               Shahid Khan and
+               Mari{\"{e}}lle Stoelinga},
+  title     = {BDDs Strike Back - Efficient Analysis of Static and Dynamic Fault
+               Trees},
+  booktitle = {{NFM}},
+  series    = {{LNCS}},
+  volume    = {13260},
+  pages     = {713--732},
+  publisher = {Springer},
+  year      = {2022},
+  doi       = {10.1007/978-3-031-06773-0\_38},
+  arxiv-id  = {2202.02829},
+  material  = {https://doi.org/10.5281/zenodo.5834212},
 }
 
-@article{HJKQV21,
+@article{HJKQV22,
```

Comment:
This key is used [here](https://github.com/moves-rwth/storm/blob/d4b7e896cddcf00e98705ca58376d27db60327a7/benchmarks.md?plain=1#L26). Could you update this?

Reply:
Done.